client/src/locales/en-US.json

```
@@ -955,7 +955,13 @@
         },
         "status": {
             "active": "Active",
-            "revoked": "Revoked"
+            "revoked": "Revoked",
+            "frozen": "Suspended"
+        },
+        "statusDescriptions": {
+            "active": "User is active.",
```

I'd be more explicit here even if it is a little redundant
```suggestion
            "active": "User is active and can access this organization.",
```